loader/convert_data.go

```diff
@@ -180,7 +180,11 @@ func parseRowValues(str []byte, table *tableInfo, columnMapping *cm.Mapping) ([]
 			row = append(row, val)
 		}
 	}
-
+	if len(table.transferCol) > 0 {
+		for _, v := range table.transferVal {
+			row = append(row, "'"+v+"'")
```

Comment:
Should i distinguish double quotes or single quotes?
In my case, dump sql during the load phase always use single quotation marks on string types.

Reply:
The single quotes is better because if user set ANSI_QUOTES SQL mode, double quotes are used for identifier.

https://dev.mysql.com/doc/refman/8.0/en/string-literals.html